backend/src/test/utils.test.ts

```diff
@@ -0,0 +1,156 @@
+import * as tap from 'tap';
+import { Event, EventRecord } from "@polkadot/types/interfaces/system";
+
+import { getParaHeadAndIdFromEvent, isIncludedParablockRecord, isInstanceOfSignedBlockJsonRpc, blockToBinary } from '../utils';
+import * as signedBlockMock from '../mocks/signedBlock.json';
+
+tap.test('Utils module', (t) => {
+  tap.test('getParaHeadAndIdFromEvent should return parablock hash and paraId', (t) => {
+    const paraId = 2088;
+    const paraHead = "0x2980ff352ab0fab013f01b3757d2fbbadf8fbdd510883cb03b111b03f358bff3";
+    const descriptor = {
+      paraId: {
+        toBigInt() {
+          return BigInt(paraId)
+        }
+      },
+      paraHead: {
+        toHex() {
+          return paraHead;
+        }
+      },
+    }
+
+    const event = {
+      data: [{ descriptor }]
+    }
+
+    const result = getParaHeadAndIdFromEvent(event as unknown as Event);
+
+    t.same(result, { blockHash: paraHead, paraId });
+
+    t.end();
+  });
+
+  tap.test('blockToBinary should convert SignedBlockJsonRpc to Buffer', (t) => {
+    const result = blockToBinary(signedBlockMock);
+
+    const expected = Buffer.from([
+      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+      0, 0, 0, 0, 0, 0, 0, 0, 0, 41, 154, 201,
+      57, 234, 71, 34, 209, 252, 247, 244, 135, 54, 117, 4,
+      14, 188, 131, 209, 68, 98, 12, 118, 32, 109, 190, 183,
+      70, 140, 243, 12, 223, 3, 23, 10, 46, 117, 151, 183,
+      183, 227, 216, 76, 5, 57, 29, 19, 154, 98, 177, 87,
+      231, 135, 134, 216, 192, 130, 242, 157, 207, 76, 17, 19,
+      20, 0, 0, 0
+    ])
```

Comment:
`Buffer.from('hex string', 'hex')` would be more readable though

Reply:
Also more cases with justifications, extrinsics and log items would be nice, but this is a start nevertheless.